scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -89,17 +89,27 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
           {{ comments_panel(institute, case, current_user, comments) }}
         </div>
       </div>
-
+        <!-- OMIM diagnoses-related code-->
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="card panel-default ">
+                <div class="panel-heading"><i class="fa fa-user-md"></i>&nbsp;Diagnoses</div>
+                <div class="card-body d-flex flex-row">
+                  <div class="col-sm-6">
+                    {{ diagnosis_phenotypes(case, institute, omim_terms) }}
```

Comment:
I think it would still be nice to add a gene or phenotype by omim number. If I try to enter with a number (phenotype) on case page I get:
```
2020-02-25 14:05:51 MansMacAirDisplay.local flask.app[80431] ERROR Exception on /cust000/643595 [GET]
Traceback (most recent call last):
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/mansmagnusson/Projects/scout/scout/server/utils.py", line 28, in decorated_function
    return render_template(template_name, **context)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/mansmagnusson/Projects/scout/scout/server/blueprints/cases/templates/cases/case.html", line 397, in top-level template code
    value="{{ button_value if button_value }}"
  File "/Users/mansmagnusson/Projects/scout/scout/server/templates/layout_bs4.html", line 1, in top-level template code
    {% extends "bootstrap4.html" %}
  File "/Users/mansmagnusson/Projects/scout/scout/server/templates/bootstrap4.html", line 18, in top-level template code
    {% block body %}
  File "/Users/mansmagnusson/Projects/scout/scout/server/templates/bootstrap4.html", line 21, in block "body"
    {% block content %}
  File "/Users/mansmagnusson/Projects/scout/scout/server/templates/layout_bs4.html", line 80, in block "content"
    {% block content_main %}{% endblock %}
  File "/Users/mansmagnusson/Projects/scout/scout/server/blueprints/cases/templates/cases/case.html", line 38, in block "content_main"
    {{ case_page() }}
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/runtime.py", line 579, in _invoke
    rv = self._func(*arguments)
  File "/Users/mansmagnusson/Projects/scout/scout/server/blueprints/cases/templates/cases/case.html", line 102, in template
    {{ diagnosis_genes(case, institute, omim_terms) }}
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/runtime.py", line 579, in _invoke
    rv = self._func(*arguments)
  File "/Users/mansmagnusson/Projects/scout/scout/server/blueprints/cases/templates/cases/phenotype.html", line 78, in template
    {{ omim_id }} - {{ omim_terms[omim_id].description }}
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/jinja2/environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: dict object has no element 102750
```

Reply:
So you want to be able to search by omim id (number), disease description and associated genes. I'll see what can be done!

Reply:
It does not necessarily have to search. If one pastes a number like `214300` it could check if it is a valid diagnosis/gene or not and then add it. If that's possible of course!

Reply:
Of course it's possible, I'll see!